### PR TITLE
Fix #4347: srw mpi logs

### LINKED ISSUE #4347: Show additional info from srw mpi log file
Provide more information from the SRW status file when running MPI jobs in the UI. Or provide a way to download the log file via a link.

### PR DESCRIPTION
**Pull request by gurhar1133**
Log file links are now available in partially coherent results, spectral flux, and coherent modes tabs.

@moellep I have run into some issues with the coherent modes tab. Not sure if reproducible, but if you run a spectral flux animation (and cancel it early once the log link becomes available) then expand the coherent modes tab you may get 'report not generated' error message in coherent modes tab or other errors.

Some errors like this were present on master branch as well. 

Maybe see if you can reproduce and I could create new issue if so?

### CODE REVIEW

**Comment by rorour** [comment]
looks good to me!

**Review by rorour — APPROVED** [review]

File: sirepo/template/srw.py [hunk 1]
```diff
@@ -456,6 +456,8 @@ def get_application_data(data, **kwargs):
 
 
 def get_data_file(run_dir, model, frame, **kwargs):
+    if frame < 0:
```
Comment (robnagler) [inline sirepo/template/srw.py:459]: This seems like an add test. Is there no other argument you could use in kwargs to make this be more explicit?

Comment (gurhar1133) [reply sirepo/template/srw.py:459]: What about passing in a string to indicate that it's a SRW log file: `'isSRWLogFile'`

Comment (robnagler) [reply sirepo/template/srw.py:459]: The call limits you at the moment to options.suffix. I think that could be a better signal than frame < 0. Look at [zgoubi.get_data_file](https://github.com/radiasoft/sirepo/blob/868fed4ad90e3c2503664528493447f89073162b/sirepo/template/zgoubi.py#L399) which abuses suffix to be the whole file name.

**Comment by gurhar1133** [comment]
Made changes to use options @robnagler. Will also work on #4428 👍

**Review by robnagler — APPROVED** [review]

**Pull request merged**